sycl/source/detail/plugin.hpp

```diff
@@ -106,6 +108,7 @@ class plugin {
 private:
   RT::PiPlugin MPlugin;
   backend MBackend;
+  std::shared_ptr<std::mutex> TracingMutex;
```

Comment:
Could you please clarify why std::mutex is wrapped with std::shared_ptr?

Reply:
Because mutex object cannot be copied.